### DISCUSSION TITLE
PVC Stuck In Attaching

### DISCUSSION THREAD

**Discussion by northboys** [longhorn/longhorn#3982] [Q&A]
Hii, My PVC Stuck In Attaching when create a lot deployment use longhorn pvc..? should I add more resources in Guaranteed Engine Manager CPU?

**Comment by innobead** [comment]
all volumes unable to attach or just some?

**Reply by northboys** [reply]
some volume..

**Comment by PhanLe1010** [comment]
Can you provide the number of PVCs and Longhorn version?

**Reply by northboys** [reply]
PVC = 20
Longhorn version = v1.2.2
Average size of PVC = 200Gi
